### PR TITLE
class_7923 -> Registries, constants

### PR DESCRIPTION
**Pull request by Benonardo**
Class name and package may not be ideal

### CODE REVIEW

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/Registries.mapping [hunk 1]
```diff
@@ -0,0 +1,71 @@
+CLASS net/minecraft/class_7923 net/minecraft/Registries
```
Comment (enbrain) [inline mappings/net/minecraft/Registries.mapping]: I think this class should be in the `net/minecraft/util/registry` package.

**Review by enbrain — COMMENTED** [review]
Can you map `field_41175`? The auto-proposed name `AIR` is extremely confusing.

File: mappings/net/minecraft/Registries.mapping [hunk 2]
```diff
@@ -0,0 +1,69 @@
+CLASS net/minecraft/class_7923 net/minecraft/util/registry/Registries
```
Comment (enbrain) [inline mappings/net/minecraft/Registries.mapping]: This file should be moved to `mappings/net/minecraft/util/registry/Registries.mapping`. I highly recommend you use Enigma when editing mappings.

Comment (Benonardo) [reply mappings/net/minecraft/Registries.mapping]: Good idea, probably won't make the same mistake twice :+1:

**Review by enbrain — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**